main.py

```diff
@@ -31,6 +31,8 @@
 
 
 bezciSobe = Flask(__name__)
+# pokus o nastaveni server_name z env kvuli plnemu url na obnovu hesla (aktualne bude mozna vracet http://0.0.0.0/
+#bezciSobe.config['SERVER_NAME'] = os.environ.get('SERVER_NAME')
```

Comment:
Při spouštění lokálně by to nemělo být potřeba. Nebo je? (Nevím.)

...ale...

Otázka je, jak se to bude chovat na Heroku. Tam by podle mě mělo být v  _Config Vars_ nastavené `SERVER_NAME` na něco jako `bezciscobe.herokuapp.com`, ale nejsem si jistý, to se bude muset vyzkoušet. V historii na dashboardu Heroku je vidět, že už tam @petrusakn něco zkoušela nastavovat :)